app/controllers/temp/mentored_tracks_controller.rb

```diff
@@ -0,0 +1,19 @@
+module Temp
```

Comment:
@iHiD I've added this temporary controller to get and update the user's mentored tracks.

Reply:
Cool. I've added `skip_before_action :verify_authenticity_token` to this as it wasn't working for me without.